frontend/src/components/eam-configurate-group/helpers/get-columns.helper.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  UsersTableHeader,
+  UsersTableAccessor,
+  StorageKey,
+} from 'common/enums/enums';
+import { Checkbox } from 'components/common/common';
+import { useState } from 'hooks/hooks';
+import { storage } from 'services/services';
```

Comment:
```suggestion
```
we can't use services directly in the component
Overall, we don't want to store this users in the localstorage